NitroxClient/GameLogic/PrecursorManager.cs

```diff
@@ -0,0 +1,21 @@
+﻿using NitroxClient.Communication.Abstract;
+using NitroxModel.Packets;
+using NitroxModel.DataStructures;
+
+namespace NitroxClient.GameLogic
+{
+    public class PrecursorManager
```

Comment:
What types of functions will this contain in the future?

Reply:
Honestly I wasnt sure, I just imagine that we have other things precursor wise that probably need syncing